README.md

```diff
@@ -46,46 +20,28 @@ ServicePulse uses [npm](https://www.npmjs.com/) and [Bower](https://bower.io/) a
 
 #### Install dependencies
 
-Install the following dependencies if you don't have them installed yet
+Install the following dependencies if you don't have them installed yet.
 
 - [Git for Windows](https://gitforwindows.org/)
 - [Node.js](https://nodejs.org/en/download/)
-  - **NOTE:** It is good practice when installing or updating node.js on your local environment to also check and make sure that the node.js version for the `ci.yml` and `release.yml` workflows match the version of your local environment. Look for the step `name: Set up Node.js`
+  - **NOTE:** It is good practice when installing or updating node.js on your local environment also to check and make sure that the node.js version for the `ci.yml` and `release.yml` workflows match the version of your local environment. Look for the step `name: Set up Node.js`
 - [Docker](https://docs.docker.com/get-docker/)
-  - **NOTE:** If running docker for windows, the current user needs to be in the docker-users local group. If your windows account is different to the installation (admin) account, then you will need to manually add your user to the group. See https://docs.docker.com/desktop/faqs/windowsfaqs/#why-do-i-see-the-docker-desktop-access-denied-error-message-when-i-try-to-start-docker-desktop
+  - **NOTE:** If running docker for Windows, the current user needs to be in the docker-users local group. If your Windows account differs from the installation (admin) account, you must manually add your user to the group. See <https://docs.docker.com/desktop/faqs/windowsfaqs/#why-do-i-see-the-docker-desktop-access-denied-error-message-when-i-try-to-start-docker-desktop>
 - Chutzpah
   - [Test Adapter for the Test Explorer](https://marketplace.visualstudio.com/items?itemName=vs-publisher-2795.ChutzpahTestAdapterfortheTestExplorer)
   - [Test Runner Context Menu Extension](https://marketplace.visualstudio.com/items?itemName=vs-publisher-2795.ChutzpahTestRunnerContextMenuExtension)
 
 ### Set development environment
 
 > [!TIP]
-> Using the `ServicePulse\src\run-dev.cmd` batch file will execute steps 1 - 3.
+> The `ServicePulse\src\run-dev.cmd` batch file will execute steps 1 - 3.
```

Comment:
```suggestion

```
We probably don't need this either since the batch file runs the steps being removed in this PR.  We could probably get rid of the run-dev.cmd file too. 

```cmd
@echo off
pushd .\ServicePulse.Host
start "Proxy" docker run -it --rm -p 1331:1331 -v %cd%/nginx.conf:/etc/nginx/nginx.conf:ro --name service-pulse-dev nginx
pushd .\vue
start "Vue" npm run dev
popd
pushd .\angular
start "Angular" npm run dev
```